rclpy/rclpy/impl/rmw_implementation_tools.py

```diff
@@ -85,23 +83,8 @@ def import_rmw_implementation():
     if __rmw_implementation_module is not None:
         return __rmw_implementation_module
     available_implementations = get_rmw_implementations()
-    if __selected_rmw_implementation is None:
-        logger = logging.getLogger('rclpy')
-        # prefer FastRTPS, otherwise first in alphabetical order
-        # the same logic is implemented in
-        # rmw_implementation_cmake/cmake/get_default_rmw_implementation.cmake
-        default_impl = 'rmw_fastrtps_cpp'
-        if default_impl in available_implementations:
-            select_rmw_implementation(default_impl)
-        elif available_implementations:
-            select_rmw_implementation(available_implementations[0])  # select the first one
-        else:
-            raise NoImplementationAvailableException()
-        logger.debug("Implicitly selecting the '{0}' rmw implementation."
-                     .format(__selected_rmw_implementation))
-    module_name = '._rclpy__{rmw_implementation}'.format(
-        rmw_implementation=__selected_rmw_implementation,
-    )
+    __selected_rmw_implementation = 'rmw_implementation'
```

Comment:
Imo we don't need two separate ways to select rmw implementation. The `RMW_IMPLEMENTATION` environment variable works across all languages. In C++ you don't have the option to choose the rmw implementation from within the running executable. Therefore I don't think this is a feature we need in Python - especially since it is not possible currently to use rmw A and then switch to rmw B within the same Python script.

I don't know if a running Python script can look at the environment variable, change the value and that change being reflected in the loaded library since I haven't tried it. But imo it is a not a use case worth supporting.

We should then make a decision on the question about selecting the rmw impl. within a running Python script in order to continue on this.